charts/azuremonitor-containers/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 7.0.0-1
 description: Helm chart for deploying Azure Monitor container monitoring agent in Kubernetes
 name: azuremonitor-containers
-version: 3.0.0
+version: 3.0.1
```

Comment:
update to 3.1.0 from 3.0.1 since we have feature changes.

Reply:
updated